styles.css

```diff
@@ -6,419 +6,1410 @@
+    margin-top: 8%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  h1 {
+    width: 450px;
+    height: 160px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+  }
+
+  .developer {
+    width: 566px;
+    height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+  }
+
+  .abilities {
+    width: 566px;
+    height: 123px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  .About-me {
+    width: auto;
+    height: 1195px;
+    left: 0;
+    top: 2680px;
+    background: #262626;
+    background-image: url(images/desktopframe.png);
+    background-repeat: no-repeat;
+    background-position: 100% 10%;
+  }
+
+  .bou-me {
+    width: 566px;
+    height: 78px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+    margin-top: 130px;
+    margin-bottom: 24px;
+  }
+
+  .abt {
+    display: grid;
+    margin-bottom: 150px;
+    margin-left: 142px;
+  }
+
+  .abt-text {
+    width: 566px;
+    height: 148px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  * .moved {
+    margin-top: 130px;
+  }
+
+  .resume {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 161px;
+    height: 48px;
+    background: #f55800;
+    color: #fff;
+    justify-content: center;
+    border: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+  }
+
+  .name:hover {
+    color: brown;
+    font-size: 70px;
+  }
+
+  /* ....... languages ...... */
+
+  #underline {
+    justify-content: center;
+    display: flex;
+    margin-bottom: 146px;
+  }
+
+  .lang-cont {
+    display: flex;
+    justify-content: center;
+    gap: 176px;
+  }
+
+  .lang-heading {
+    display: flex;
+    align-items: baseline;
+    font-family: 'InaiMathi', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    gap: 10px;
+    margin-bottom: 20px;
+    margin-top: 20px;
+  }
+
+  .langlist {
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+  }
+
+  .jss {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+    list-style: none;
+  }
+
+  form {
+    display: flex;
+    align-items: flex-start;
+    justify-content: center;
+    gap: 30px;
+    height: 590px;
+  }
+
```

Comment:
- [ ] Nice work implementing the form it looks really good:clap: However let's stick to the design as much as possible (e.g., font, colors, images, text, margins), I noticed that your `input` and `textarea` is a little bit different from the design (see image).

### Your design
![Screenshot from 2023-03-29 14-04-28](https://user-images.githubusercontent.com/91142432/228568500-1a2957ac-0f14-4db0-a112-17e827848d52.png)

### Figma's design
![Screenshot from 2023-03-29 13-49-56](https://user-images.githubusercontent.com/91142432/228568679-3a19543f-c6de-4e46-a899-d410abe0cda9.png)

Reply:
I have made the correction, thank you.